### PR TITLE
feat(cards): apply new specifications

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Apply [new component specifications](https://webgate.ec.europa.eu/CITnet/confluence/x/nZNOMQ)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 8953b4ef98ccebd14bac36c96873c1bd1a48d548

https://deploy-preview-829--europa-component-library.netlify.com

File: src/systems/ec/implementations/react/packages/card/stories/Template.jsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  };
+  const title6 = {
+    variant: 'standalone',
+    label:
+      'France, Europe and development aid : from the treaties of Rome to the present day',
+    href: '',
+    level: '3',
+  };
+
+  return (
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/card/stories/Template.jsx]: If you're using an `ecl-row`, please wrap the template within an `ecl-container` and remove the `margin: 0` from the `ul`

File: src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  $max-width: 21.75rem,
+  $min-width: 20rem,
+  $background-card: $ecl-color-grey-5,
+  $border-color-card: $ecl-color-grey-10,
+  $color-meta: $ecl-color-grey-75,
+  $color-description: $ecl-color-grey-100,
+  $color-info: $ecl-color-grey-75
+) {
+  .ecl-card {
+    background-color: $background-card;
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss]: `box-sizing: border-box;` is missing

File: src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    border: 1px solid $border-color-card;
+    margin: 0;
+    max-width: $max-width;
+    min-width: $min-width;
+    padding-left: $ecl-spacing-s;
+    padding-right: $ecl-spacing-s;
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss]: the paddings should take the border into account

File: src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  }
+
+  .ecl-card__image {
+    background-size: cover;
+    display: block;
+    height: auto;
+    margin-top: $ecl-spacing-s;
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss]: the margin should take the border into account

File: src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+  }
+
+  /*
+   * Tiles
+   */
+  .ecl-card--tile {
+    .ecl-card__title {
+      font: $ecl-font-prolonged-l;
+      font-weight: bold;
+      margin-top: $ecl-spacing-l;
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss]: don't forget to take the border into account

File: src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+  /*
+   * Spacing adjustment
+   */
+  .ecl-card__meta + .ecl-card__title {
+    margin-top: $ecl-spacing-xs;
+  }
+
+  .ecl-card > :last-child {
+    margin-bottom: $ecl-spacing-s;
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss]: don't forget the border

File: src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+    margin-bottom: $ecl-spacing-l;
+  }
+
+  .ecl-card__title:last-child {
+    margin-bottom: $ecl-spacing-m;
+  }
+}
+
+// Use mixin
+@include exports('eu-component-card') {
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card.scss]: `'eu-component-card'` -> `'ec-component-card'`

File: src/systems/ec/implementations/vanilla/packages/ec-component-card/package.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@ecl/ec-component-card",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.0.0-alpha.0",
+  "description": "EU Card",
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-card/package.json]: -> `ECL EC Card`
btw, doesn't it depend on the link component?

File: src/systems/ec/implementations/vanilla/packages/ec-preset-full/_imports.scss [hunk 9]
```diff
@@ -58,6 +58,7 @@
 // Molecules
 @import '@ecl/ec-component-hero-banner/ec-component-hero-banner';
 @import '@ecl/ec-component-breadcrumb/ec-component-breadcrumb';
+@import '@ecl/ec-component-card/ec-component-card';
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-preset-full/_imports.scss]: Don't forget to also add the component to the preset's package.json dependencies

File: src/systems/ec/specs/components/card/package.json [hunk 10]
```diff
@@ -3,7 +3,7 @@
   "author": "European Commission",
   "license": "EUPL-1.1",
   "version": "2.0.0-alpha.0",
-  "description": "ECL Card Specs",
+  "description": "ECL EC Logo Specs",
```
Comment (yhuard) [inline src/systems/ec/specs/components/card/package.json]: why "Logo"?

**Review by yhuard — CHANGES_REQUESTED** [review]
reviewed EC, most of the remarks are also valid for EU

File: src/systems/ec/specs/components/card/docs/code.mdx [hunk 11]
```diff
@@ -0,0 +1,17 @@
+import Playground from '@ecl/website-components/Playground'
+import Card from '@ecl/ec-react-component-card/Card'
+
+import demoContentCard from '../demo/data--card';
+import demoContentTile from '../demo/data--tile';
+
+## Card
+
+<Playground playgroundLink="/storybook/ec/index.html?selectedKind=Card&selectedStory=card&stories=0">
```
Comment (yhuard) [inline src/systems/ec/specs/components/card/docs/code.mdx]: You can use the new syntax here (check https://github.com/ec-europa/europa-component-library/blob/next-v2/src/systems/ec/specs/components/hero-banner/docs/code.mdx#L13)

File: src/systems/ec/implementations/vanilla/packages/ec-component-card/package.json [hunk 12]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ecl/ec-component-card",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.0.0-alpha.0",
+  "description": "ECL EC Card",
+  "style": "ec-component-card.scss",
+  "dependencies": {
+    "@ecl/ec-base": "^2.0.0-alpha.0",
+    "@ecl/ec-component-link": "^2.0.0-alpha.0",
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-card/package.json]: if you check the imports in sass for those dependencies, it's even better :)

**Review by yhuard — CHANGES_REQUESTED** [review]
2 (optional) enhancements

**Review by yhuard — APPROVED** [review]

**Pull request merged**